### PR TITLE
feat: Add new tutorial steps related to social features

### PR DESCRIPTION
**Pull request by sandrade-dcl**
- [x] Add a new tutorial step (tooltip) for the **Users Around** list.

image

- [x] Add a new tutorial step (tooltip) for the **social features** groups in the taskbar (voice chat, world chat and friends).

image

- [x] Create a **new tutorial path** for users that already saw the tutorial (this path should be executed only one time per user). In this path the user should view the tooltip for the **Users Around** list but it will be automatically hidden after 5 seconds.
**NOTE**: _The implementation of this new path is something TEMPORAL until we refactorize the tutorial system to make it compatible with incremental features._

image

**TESTING URL**: https://play.decentraland.zone/branch/feat/New-tutorial-step-for-the-Users-Around-list/index.html?ENV=org

**NOTE**: Add the param `&RESET_TUTORIAL` in the url if you need to restart the tutorial flow.

### CODE REVIEW

File: unity-client/Assets/Tutorial/Scripts/TutorialController.cs [hunk 1]
```diff
@@ -171,6 +178,11 @@ public void SetTutorialEnabled(string fromDeepLink)
             SetupTutorial(fromDeepLink, TutorialType.Initital);
         }
 
+        public void SetTutorialEnabledForUsersThatAlreadyDidTheTutorial()
+        {
+            SetupTutorial(false.ToString(), TutorialType.Initital, true);
```
Comment (BrianAmadori) [inline unity-client/Assets/Tutorial/Scripts/TutorialController.cs]:
```suggestion
            SetupTutorial(false.ToString(), TutorialType.Initial, true);
```

**Review by BrianAmadori — COMMENTED** [review]
Looks good! It's ok to show the new step every time the user goes into explorer? It can be annoying.

File: kernel/packages/unity-interface/UnityInterface.ts [hunk 2]
```diff
@@ -306,6 +306,10 @@ export class UnityInterface {
     this.gameInstance.SendMessage('TutorialController', 'SetTutorialEnabled', JSON.stringify(fromDeepLink))
   }
 
+  public SetTutorialEnabledForUsersThatAlreadyDidTheTutorial() {
+    this.gameInstance.SendMessage('TutorialController', 'SetTutorialEnabledForUsersThatAlreadyDidTheTutorial')
```
Comment (AjimenezDCL) [inline kernel/packages/unity-interface/UnityInterface.ts:310]: we really need a better way for this in the future. Shall we create a ticket so we dont forget?

**Review by AjimenezDCL — APPROVED** [review]
Approving with some concerns on the approach, but since the release is pushing, let's solve it later.

File: kernel/packages/unity-interface/UnityInterface.ts (hunk 2 above)
Comment (pbosio) [reply kernel/packages/unity-interface/UnityInterface.ts:310]: I was about to write the same comment 😂
I don't like this approach very much, I have an idea to doing it more scalable but we are now in a hurry, so we can revisit it later

**Comment by sandrade-dcl** [comment]
> Looks good! It's ok to show the new step every time the user goes into explorer? It can be annoying.

Yes, it will be always showed but it automatically hides after 5 seconds.

File: unity-client/Assets/Tutorial/Scripts/TutorialController.cs (hunk 1 above)
Comment (sandrade-dcl) [reply unity-client/Assets/Tutorial/Scripts/TutorialController.cs]: Done

**Comment by pbosio** [comment]
> > Looks good! It's ok to show the new step every time the user goes into explorer? It can be annoying.
> 
> Yes, it will be always showed but it automatically hides after 5 seconds.

We could add something like `UsersAroundListStep = 512` to the `TutorialFinishStep` and add it to the tutorialflag of the profile to avoid showing it again

File: kernel/packages/unity-interface/UnityInterface.ts (hunk 2 above)
Comment (sandrade-dcl) [reply kernel/packages/unity-interface/UnityInterface.ts:310]: Yes, I agree. In this case we have needed to apply these changes on the tutorial flow a day before the release and it has been the quicker way. I will annote this for a future improvement.

**Review by pravusjif — APPROVED** [review]

**Review by pbosio — APPROVED** [review]

**Pull request merged**